sdk/maps/Azure.Maps.Routing/src/Generated/RoutingModelFactory.cs

```diff
@@ -12,7 +12,7 @@
 namespace Azure.Maps.Routing.Models
 {
     /// <summary> Model factory for models. </summary>
-    public static partial class RouteModelFactory
+    public static partial class RoutingModelFactory
```

Comment:
name changed to MapsRoutingModelFactory to align with `Azure.Maps.Rendering`
This change should be OK since this package is beta